frontend/mulighetsrommet-veileder-flate/cypress/support/e2e.js

```diff
@@ -55,17 +68,22 @@ before('Start server', () => {
   cy.getByTestId('tiltakstype-oversikt').children().should('have.length.greaterThan', 1);
 });
 
+Cypress.Commands.add('resetSide', () => {
+  cy.visit('/');
+});
+
 Cypress.Commands.add('getByTestId', (selector, ...args) => {
   return cy.get(`[data-testid=${selector}]`, ...args);
 });
 
 Cypress.Commands.add('velgFilter', filternavn => {
-  cy.getByTestId(`filter_checkbox_${filternavn}`).should('not.be.checked').click().should('be.checked');
+  cy.getByTestId(`filter_checkbox_${filternavn}`).click();
+  cy.getByTestId(`filter_checkbox_${filternavn}`).should('be.checked');
   cy.getByTestId(`filtertag_${filternavn}`).should('exist');
 });
 
 Cypress.Commands.add('fjernFilter', filternavn => {
-  cy.getByTestId(`filter_checkbox_${filternavn}`).should('be.checked').click().should('not.be.checked');
+  cy.getByTestId(`filtertag_lukkeknapp_${filternavn}`).click();
   cy.getByTestId(`filtertag_${filternavn}`).should('not.exist');
 });
```

Comment:
Siden vi nå har en blanding av checkbox og radiobutton er det bedre at vi fjerner filter fra filtertags og ikke fra selve filteret. Om man prøver å fjerne et filter for innsatsgruppe så vil ikke det gå fordi klikk på en radioknapp som allerede er klikket på ikke gjør noen forskjell